service_capacity_modeling/models/org/netflix/cassandra.py

```diff
@@ -64,20 +64,26 @@ def _write_buffer_gib_zone(
 
 def _estimate_cassandra_requirement(
     instance: Instance,
+    current_instance: Instance,
```

Comment:
This is where I'd pass the existing plan and then use that for e.g. current sizing